### PR TITLE
Doku for release2.1.0crp

### LINKED ISSUE #77: Update Year in License
For the next release:

[LICENSE](https://github.com/ComputationalRadiationPhysics/mallocMC/blob/15730e4b79b0cda496ad81d4998cbc3ea6cf9fc9/LICENSE#L12-L13) Update "our" years.

### PR DESCRIPTION
**Pull request by slizzered**
- Update the Changelog
- Update `Usage.md` to reflect interface changes introduced by #73
- also closes #77

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   Copyright (C) 2012 Institute for Computer Graphics and Vision,
                      Graz University of Technology
-  Copyright (C) 2014 Institute of Radiation Physics,
+  Copyright (C) 2015 Institute of Radiation Physics,
```
Comment (ax3l) [inline LICENSE]: uh, that should be 2014-2015 actually!

**Comment by ax3l** [comment]
just realized: version bump is missing

**Comment by ax3l** [comment]
pls update the release date to today at the same time

**Comment by slizzered** [comment]
ok, will open a new PR, I guess

**Comment by slizzered** [comment]
... and one day I will write some checklist on what to do prior to each release...

**Comment by ax3l** [comment]
just add a small list in the wiki :)

**Comment by ax3l** [comment]
but yes, forgetting to bump the version number is probably the worst thing that can happen :D usually requires a follow-up release ^^

**Comment by slizzered** [comment]
new PR is ready :D

**Pull request merged**